src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -617,22 +619,113 @@ object Helpers extends SparkSessionWrapper {
    *                        or direct access via s3:// or abfss:// or dbfs:/mnt/ etc.
    * @return
    */
+//  def registerRemoteOverwatchIntoLocalMetastore(
+//                                                 remoteWorkspace: Workspace,
+//                                                 localDataTarget: DataTarget
+//                                               ): Seq[WorkspaceMetastoreRegistrationReport] = {
+//
+//    val newConfigParams = remoteWorkspace.getConfig.inputConfig.copy(dataTarget = Some(localDataTarget))
+//    val newConfigArgs = JsonUtils.objToJson(newConfigParams).compactString
+//    val localTempWorkspace = Initializer(newConfigArgs, disableValidations = true)
+//    val registrationReport = localTempWorkspace.addToMetastore()
+//    val b = Bronze(localTempWorkspace, suppressReport = true, suppressStaticDatasets = true)
+//    val g = Gold(localTempWorkspace, suppressReport = true, suppressStaticDatasets = true)
+//
+//    b.refreshViews()
+//    g.refreshViews()
+//    registrationReport
+//
+//  }
   def registerRemoteOverwatchIntoLocalMetastore(
-                                                 remoteWorkspace: Workspace,
-                                                 localDataTarget: DataTarget
-                                               ): Seq[WorkspaceMetastoreRegistrationReport] = {
+                                                 remoteStoragePrefix: String,
```

Comment:
function needs comments for functionality as well as for each parameter. For example -- what is `remoteETLDataPathPrefixOverride` supposed to be? storagePrefix? storagePrefix/global_share_custom etc? Give description plus examples using common Runner Notebook nomenclature.

Reply:
and storagePrefix should truly be the storage prefix not the data prefix.